custom_components/nexa_bridge_x/entities.py

```diff
@@ -77,8 +77,8 @@ def __init__(self, node: NexaNode, coordinator: CoordinatorEntity) -> None:
 
 class NexaDimmerEntity(NexaNodeEntity, LightEntity):
     """Entity for light"""
-    _attr_color_mode = ColorMode.BRIGHTNESS
-    _attr_supported_color_modes = {ColorMode.ONOFF, ColorMode.BRIGHTNESS}
+    _attr_color_mode = ColorMode.ONOFF
+    _attr_supported_color_modes = {ColorMode.BRIGHTNESS}
```

Comment:
These changes needs to go into a separate PR and then be verified. Along with dumps of the nodes you have that required these modifications.

Reply:
Ideally an issue should be created if there is some kind of issue or required change to underlying entities, etc.